tests/kernels/test_cutlass.py

```diff
@@ -9,13 +9,21 @@
 
 from vllm import _custom_ops as ops
 from vllm.platforms import current_platform
-
-CUDA_DEVICES = [
-    f"cuda:{i}" for i in range(1 if torch.cuda.device_count() == 1 else 2)
-]
+from vllm.utils import is_cpu
```

Comment:
Like the previous mention on overloading the name `cutlass_scaled_mm`, I think it makes sense to duplicate a separate kernel test for CPU gemm

I appreciate the kernel testing but I would like to see an e2e test making sure model loading, weight packing, etc works since this PR plugs into compressed-tensors - just loading the model and generating a few tokens should be sufficient

Reply:
We can use cases from ```test_compressed_tensors.py``` and execute E2E tests to verify the feature workflow on the CPU backend. 

The kernel tests will be keep internally and executed regularly.